src/YGFrame.cc

```diff
@@ -134,6 +134,8 @@ class YGCheckBoxFrame : public YCheckBoxFrame, public YGBaseFrame
 
 		setLabel (label);
 		setValue (checked);
+    doSetEnabled( checked );
```

Comment:
Please notice that this behaviour should be inverted if the `invertAutoEnable()` flag is set (see https://github.com/libyui/libyui/blob/master/libyui/src/YCheckBoxFrame.h#L105).

So AFAICS this should rather be

```C++
doSetEnabled( checked && ! invertAutoEnable() )
```

(Please check if this is correct)

Reply:
Interesting, i just changed the code and didn't know about that property. Thanks for the hint, I'll do some test more following your suggestion :) Thanks, will call you again after.

Reply:
Sorry for the delay. I've experienced that that property does not work properly at start-up with your suggestion. I'm investigating. BTW i've exprienced it also using Qt. I will provide e simple test example and open a related issue if after testing again that is confirmed

Reply:
@shundhammer i've fixed the autoenable/inverautoenable behavior see https://github.com/libyui/libyui /issues/45 and issue #81